Gems/PhysXDebug/Code/Source/SystemComponent.cpp

```diff
@@ -468,16 +468,25 @@ namespace PhysXDebug
         RenderBuffers();
     }
 
+    AZ::Vector3 GetViewCameraPosition()
+    {
+        using namespace Camera;
```

Comment:
nit: We generally advise to using directives like this, instead prefer using declarations like `using Camera::ActiveCameraRequestBus;` (there's a whole load of great posts on the Abseil blog about why `using namespace` can cause loads of problems - https://abseil.io/tips/153)